contracts/clearing/TradingApp.sol

```diff
@@ -113,53 +113,38 @@ contract TradingApp is IForceMoveApp {
 		ITradingTypes.Settlement memory settlement,
 		RecoveredVariablePart[] calldata proof
 	) internal pure {
+		VariablePart memory currProof = proof[0].variablePart;
+		VariablePart memory nextProof = proof[1].variablePart;
+		require(
+			currProof.turnNum >= 2 && nextProof.turnNum >= 2,
+			'only prefund and postfund can have turn number < 2'
+		);
+
```

Comment:
Actually, we can fully omit this check as the first iteration below will check `prevTurnNum + 1 == currProof.turnNum` and indeed `proof[0].turnNum = 1 + 1 = 2`

Reply:
What about starting the loop from `i = 2` instead?